data/core/units/orcs/Shaman.cfg

```diff
@@ -0,0 +1,99 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Orcish Shaman
+    name=_"Orcish Shaman"
+    race=orc
+    image="units/orcs/shaman.png"
+    profile="portraits/orcs/shaman.webp"
+    hitpoints=23
+    movement_type=orcishfoot
+    [defense]
+        village=30
+    [/defense]
+    movement=5
+    experience=120
+    level=2
+    alignment=chaotic
+    advances_to=Orcish Sorcerer
+    cost=21
```

Comment:
Level 2, cost should be higher. maybe like 28-30?

Reply:
raised to 28 in 28ef66c